connector_onpay/views/product_template_view.xml

```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="view_product_template_onpay_inherit" model="ir.ui.view">
+        <field name="name">product.template.onpay</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
```

Comment:
```suggestion
        <field name="inherit_id" ref="hr_expense.product_product_expense_form_view"/>
```


@Chandresh-SerpentCS This view is not used in Expenses > Configuration > Expenses Products